server/mock/store.go

```diff
@@ -6,15 +6,16 @@ import (
 	protoio "github.com/gogo/protobuf/io"
 	dbm "github.com/tendermint/tm-db"
 
-	snapshottypes "github.com/cosmos/cosmos-sdk/snapshots/types"
-	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	pruningtypes "github.com/cosmos/cosmos-sdk/pruning/types"
+	snapshotypes "github.com/cosmos/cosmos-sdk/snapshots/types"
+	storeTypes "github.com/cosmos/cosmos-sdk/store/types"
```

Comment:
Why this odd camelCasing yet all the other imports are snakecase? But even more this creates unnecessary edits to the diff.

Reply:
fixed